client/src/components/Applications/AppLayout/AppLayout.js

```diff
@@ -24,7 +25,10 @@ const AppsLayout = () => {
         <>
           {Boolean(Object.keys(categoryDict).length) && <AppBrowser />}
           {!params.appId && (
-            <AppPlaceholder apps={Boolean(Object.keys(categoryDict).length)} />
+            <AppPlaceholder
+              apps={Boolean(Object.keys(categoryDict).length)}
+              error={error}
```

Comment:
any reason not to show the error message here in AppLayout.js instead of in AppPlaceholder?

Reply:
Also regardless of where the error message ends up, can you add a related unit test to check for that?